src/components/ModuleNav.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <section class="left-section">
+    <router-link to="/mod-1" class="module">Module 1</router-link>
+    <router-link to="/mod-2" class="module">Module 2</router-link>
+    <router-link to="/mod-3" class="module">Module 3</router-link>
+    <router-link to="/mod-4" class="module">Module 4</router-link>
+  </section>
+</template>
+
+<script>
+</script>
```

Comment:
I noticed these `<script>` tags don't have anything inside them. Does anyone know if it's best practice to have these here even if there's not content for them? Or should they be deleted?

Reply:
Nice catch! I think we will delete them going forward!